lib/cli/commands/build.js

```diff
@@ -52,28 +53,31 @@ optimized output compatible with Node 6.
     });
   }
 
-  build(dirs = this.dirs) {
-    if (!dirs) {
-      dirs = this.dirs = [ 'app', 'config', 'blueprints' ];
+  build(globs = this.globs) {
+    if (!globs) {
+      globs = this.globs = [ 'app/**/*', 'config/**/*', 'blueprints/*/index.js' ];
```

Comment:
this is starting to feel like it might be time to broccoli-ify this build process. We could support something similar to `ember-cli-build.js`, where the userland app can modify it's build process by overriding various hooks. Thoughts?